src/env/prod/terraform.tfvars

```diff
@@ -0,0 +1,171 @@
+apim_notification_sender_email = "info@pagopa.it"
+apim_publisher_email           = "io-operations@pagopa.it"
+apim_publisher_name            = "PagoPa Centro Stella DEV"
+apim_sku                       = "Developer_1"
+app_gateway_host_name          = "api.cstar-dev.pagopa.it"
```

Comment:
`app_gateway_host_name` probably is no more needed. Remove also the var from variables.tf

Reply:
Done. Thank you.